lib/vdsm/tool/__init__.py

```diff
@@ -20,9 +20,39 @@
 from __future__ import absolute_import
 from __future__ import division
 import functools
+import logging
 import os
 
 
+LOGGER_NAME = "vdsm-tool"
+
+
+def setup_logging(log_file, verbosity, append):
+    level = logging.DEBUG if verbosity else logging.INFO
+    root_logger = logging.getLogger(LOGGER_NAME)
+    root_logger.setLevel(level)
+
+    if root_logger.handlers:
+        return root_logger
+
+    if log_file is not None:
+        mode = append and 'a' or 'w'
+        file_handler = logging.FileHandler(log_file, mode=mode)
+        file_handler.setFormatter(
+            logging.Formatter(
+                "%(asctime)s [%(levelname)-7s] %(message)s",
+                "%m/%d/%Y %I:%M:%S %p"))
+        # File logging always at DEBUG level
+        file_handler.setLevel(logging.DEBUG)
+        root_logger.addHandler(file_handler)
+
+    handler = logging.StreamHandler()
+    handler.setFormatter(logging.Formatter("%(levelname)-7s | %(message)s"))
+    root_logger.addHandler(handler)
```

Comment:
Why is stdout logging enabled now when file logging is enabled?

Reply:
I can see how this may be a questionable change.

The reasoning in my head was that as a user, probably is nice to keep the logs in the console even with file logging enabled. File logging will provide more detailed logs, to share and parse, but if something goes wrong we will still get the messages in the console without having to open the log file.

Again, I can see also how this may be questionable, depending on how/when file logging is used, so I can revert it.

Reply:
OK, makes sense, as long as it doesn't break anything that might be looking at console errors when file logging is enabled.

Please add the reasoning behind this change to the commit message.